jenkins/install_jenkins.sh

```diff
@@ -198,8 +198,21 @@ sudo apt-get update --yes
 sudo apt-get install openjdk-8-jre openjdk-8-jre-headless openjdk-8-jdk --yes
 
 #install jenkins
-sudo apt-get install jenkins --yes
-sudo apt-get install jenkins --yes # sometime the first apt-get install jenkins command fails, so we try it twice
+if [[ ${jenkins_release_type} == 'verified' ]]; then
+  jenkins_version=$(curl --silent "${artifacts_location}/jenkins/jenkins-verified-ver")
```

Comment:
That's not a good idea. The odds of someone accidentally breaking master are very high